### DISCUSSION TITLE
FIXED: Hierarchy field names are case sensitive

### DISCUSSION THREAD

**Discussion by fieldrabbit21** [SkepticMystic/breadcrumbs#109] [Q&A]
My issue is that the List/Matrix remains empty.

I have installed version 0.10.46 of Breadcrumb. In this version I could not find the setting for metadata fileds (parent, child, sibling)

I have manually entered the relationship at the top.
---
parent: [[somenote]]
---
 Can you please assist?

**Comment by SkepticMystic** [comment]
Hi @fieldrabbit21 :) 
Have you not updated Breadcrumbs in a while? Things have been changing very quickly!
I think the part that you'll need to change is the new settings for hierarchies. You can read more about the hierarchies feature [here](https://github.com/SkepticMystic/breadcrumbs/wiki/Hierarchies). Let me know if you run into trouble, or need some help getting things set up 😄

**Comment by fieldrabbit21** [comment]
Hi @SkepticMystic,
Thanks for feedback. 

I am new to Obsidian and Breadcrumb. I downloaded Breadcrumb from inside Obsidian. I also did a "Check for updates", but Obsidian says I have the latest version. Is it not the latest version?

**Reply by SkepticMystic** [reply]
I see! Welcome to Obsidian :)
You should be on the latest version then, yes.
Can you send me a screenshot of your settings for `Hierarchies`, something like this:

![image](https://user-images.githubusercontent.com/70717676/130441861-570e6ba7-24ee-4b42-9fca-df6ee08ffc6f.png)

**Comment by fieldrabbit21** [comment]
![image](https://user-images.githubusercontent.com/20489448/130442051-d9d916b3-4b6d-4f89-998d-9e1a42bb7ce8.png)

**Reply by SkepticMystic** [reply]
Looks good. So you've added some metadata to your notes then, something like `parent: [[somenote]]`

1. Are you using dataview? You don't have to, but it makes things different.
2. Can you send the markdown content of one of the note's you put breadcrumbs in, for example:

```md
---
parent: [[somenote]]
---

# Heading

and stuff
```

3. Press Refresh Index and see if anything appears

**Comment by fieldrabbit21** [comment]
OK, it's working now.
Everything was correct, except for the casing, **P**arent instead of parent

Thanks for your assistance @SkepticMystic !

**Reply by SkepticMystic** [reply]
Awesome, good to hear!
Let me know if I can help with anything else :)

**Comment by jd-santos** [comment]
@SkepticMystic Sorry to bump a fixed post, but I wanted to add that I had given up on Breadcrumbs for months because I just couldn't get it to work. I just realized tonight I had this exact issue, using parent instead of Parent. 

I don't think it's worth opening a full issue, but maybe the helper text could mention the case sensitivity or the uncapitalized versions should also be included by default.

**Comment by michaelpporter** [comment]
Closing — the author marked this FIXED in the title. Reopen if it resurfaces in v4.
